srcpkgs/ufw/template

```diff
@@ -3,9 +3,8 @@ pkgname=ufw
 version=0.35
 revision=5
 hostmakedepends="python iptables"
-depends="${hostmakedepends}"
-conf_files="/etc/default/ufw /etc/ufw/ufw.conf"
```

Comment:
Please could you explain why you removed conf_files= ?

Reply:
It's a dpulicate, if you look at the whole file, below there's
```
conf_files="
 /etc/ufw/after.init
 /etc/ufw/after.rules
 /etc/ufw/after6.rules
 /etc/ufw/before.init
 /etc/ufw/before.rules
 /etc/ufw/before6.rules
 /etc/ufw/sysctl.conf
 /etc/ufw/ufw.conf
 /etc/ufw/user.rules
 /etc/ufw/user6.rules
 /etc/ufw/applications.d/*"
```

Reply:
ah yes, sorry... I should open my eyes.